pkg/fuzzy/aws_translate.go

```diff
@@ -112,15 +117,9 @@ func (a *AWSTranslate) Translate(ctx context.Context, messages *model.Messages)
 	// skip locale part if region is not a country,
 	// AWS only supports ISO 3166 2-digit country codes.
 	// https://docs.aws.amazon.com/translate/latest/dg/what-is-languages.html
-	targetLanguage := messages.Language.String()
```

Comment:
You removed the code and left the comment that applies to the removed code. Why? :)